### ISSUE TITLE
3D plot doesn't show

### ISSUE THREAD

**Issue by gagern** [sagemathinc/smc#15]
```
line3d(((1,0,0),(0,0,1)))
```

displays a 3d line, but

```
line3d(((1,0,0),(0,0,1))).show()
```

only prints

```
Graphics3d Object
```

This is particularly problematic in combination with `@interact`, since returning a 3D scene from such a function doesn't cause it to get printed.

**Comment by haraldschilly** [comment]
Please try `show(<object>)` instead of `<object>.show()`.

**Comment by gagern** [comment]
Looks a lot better, thank you. It would be nice if the two notations could be made equivalent. Shouldn't this be possible, with the rich representations framework? After all, `Graphics3d.show()` essentially boils down to `get_display_manager().display_immediately(self, **kwds)` so at least as far as I understand things, the display manager should have full control over what's going on there.

**Comment by haraldschilly** [comment]
@gagern you are right, and I remember that there has been a progress at sage days 64.25. from my POV, it seems that this is not that easy... it's maybe this ticket: http://trac.sagemath.org/ticket/18510

**Comment by gagern** [comment]
Perhaps you could monkey-patch this in without going through the whole `_rich_repr_` magic. Something like this:

```py
original = DisplayManager.display_immediately
def monkeyPatch(self, obj, *args, **kwds):
    if (isinstance(obj, Graphics3d)):
        show(obj, *args, **kwds)
    else:
        original(self, obj, *args, **kwds)
DisplayManager.display_immediately = monkeyPatch
```

**Comment by haraldschilly** [comment]
Well, I don't know. /cc @williamstein

**Comment by williamstein** [comment]
This is now fixed due to sage-6.9 having a hook to allow patching show.   

https://github.com/sagemathinc/smc/commit/647d5689382111dfe84b0048c18726b2bde82dc7

**Comment by williamstein** [comment]
Working example: https://cloud.sagemath.com/projects/4a5f0542-5873-4eed-a85c-a18c706e8bcd/files/support/2015-10-10-095226-3d-show.sagews
